utils/nethttpadaptor/nethttpadaptor.go

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
This file was previously in components-contrib, where it’s not needed anymore. Moved here as it’s needed until fasthttp is completely removed